index.bs

```diff
@@ -95,13 +95,21 @@ Examples {#examples}
 --------------------
 
 ```js
+let userControlledInput = "&lt;img src=x onerror=alert(1)//&gt;";
+
+// Create a DocumentFragment from unsanitized input:
+let s = new Sanitizer();
+let sanitizedFragment = s.sanitize(userControlledInput);
+
+// Replace an element's content from unsanitized input:
+element.innerText = "";
+element.appendChild(s.sanitize(userControlledInput));
```

Comment:
Nit: Instead of 
```
element.innerText = "";
element.appendChild(s.sanitize(userControlledInput));
```
You could just do `element.replaceChildren(s.sanitize(userControlledInput));`

Reply:
Done.